migrations/versions/0462_add_pinpoint_fields.py

```diff
@@ -0,0 +1,42 @@
+"""
+
+Revision ID: 0462_add_pinpoint_fields
+Revises: 0461_add_rtl_column_templates
+Create Date: 2024-10-15 18:24:22.926597
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+revision = "0462_add_pinpoint_fields"
+down_revision = "0461_add_rtl_column_templates"
+
+
+def upgrade():
+    op.add_column("notifications", sa.Column("sms_total_message_price", sa.Float(), nullable=True))
+    op.add_column("notifications", sa.Column("sms_total_carrier_fee", sa.Float(), nullable=True))
+    op.add_column("notifications", sa.Column("sms_iso_country_code", sa.VARCHAR(), nullable=True))
+    op.add_column("notifications", sa.Column("sms_carrier_name", sa.VARCHAR(), nullable=True))
+    op.add_column("notifications", sa.Column("sms_message_encoding", sa.VARCHAR(), nullable=True))
+    op.add_column("notifications", sa.Column("sms_origination_phone_number", sa.VARCHAR(), nullable=True))
+    op.add_column("notification_history", sa.Column("sms_total_message_price", sa.Float(), nullable=True))
+    op.add_column("notification_history", sa.Column("sms_total_carrier_fee", sa.Float(), nullable=True))
+    op.add_column("notification_history", sa.Column("sms_iso_country_code", sa.VARCHAR(), nullable=True))
+    op.add_column("notification_history", sa.Column("sms_carrier_name", sa.VARCHAR(), nullable=True))
+    op.add_column("notification_history", sa.Column("sms_message_encoding", sa.VARCHAR(), nullable=True))
+    op.add_column("notification_history", sa.Column("sms_origination_phone_number", sa.VARCHAR(), nullable=True))
```

Comment:
We should update the batch process too. I wonder if we can do it in a different PR or if that will have unintended side-effects when the archival jobs will run. Definitely something to test.

Reply:
tested locally
- sent an SMS, 
- edited the database to give those fields values since we don't process the AWS receipts locally
- also set the dates for the sms in the database to be a month ago
- edited the schedule for the archiving task `delete-sms-notifications` so it would be scheduled in a couple minutes
- reran celery locally and waited for it to run
- checked the database - sms with correct fields now in history table, and removed from notifications table 🎉